source/dub/dependency.d

```diff
@@ -706,7 +774,7 @@ struct Version {
 		Note that branches are always considered pre-release versions.
 	*/
 	@property bool isPreRelease() const {
-		if (isBranch) return true;
+		if (isBranch || isGit) return true;
```

Comment:
Why should `isGit` necessarily imply a pre-release?  What if the git version spec points to a commit with a release tag?